### PR TITLE
Add GHA to Helm Charts

### PR DESCRIPTION
**Pull request by gab-arrobo**
Currently, this PR is set as draft because `secrets` need to be set up before merging it.

@dafATonf, please your help configuring the secrets either at the repo level or organization level

### CODE REVIEW

**Review by SeanCondon — COMMENTED** [review]
nice. i will wait to review when you move it out of draft

**Comment by gab-arrobo** [comment]
> nice. i will wait to review when you move it out of draft

Feel free to review it. The PR is ready and the only reason why I have it as `draft` is to prevent the PR from being merged before the secrets are in place.

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+# Copyright 2024 Intel Corporation
+
+name: Tag/Publish Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
```
Comment (SeanCondon) [inline .github/workflows/push.yml:11]: this looks great - I presume you're reusing from sdran-helm-charts

**Review by SeanCondon — APPROVED** [review]

File: .github/workflows/push.yml (hunk 1 above)
Comment (gab-arrobo) [reply .github/workflows/push.yml:11]: No, the `sdran-helm-charts` does not have GHAs. I am reusing the GHAs I created for the `sdcore-helm-charts`

**Pull request merged**